libbeat/kibana/fields_transformer_test.go

```diff
@@ -97,14 +98,75 @@ func TestMissingVersion(t *testing.T) {
 }
 
 func TestDuplicateField(t *testing.T) {
+	testCases := []struct {
+		commonFields []common.Field
+	}{
+		// type change
+		{commonFields: []common.Field{
+			{Name: "context", Path: "something"},
+			{Name: "context", Path: "something", Type: "date"},
+		}},
+		// missing overwrite
+		{commonFields: []common.Field{
+			{Name: "context", Path: "something"},
+			{Name: "context", Path: "something"},
+		}},
+		// missing overwrite in source
+		{commonFields: []common.Field{
+			{Name: "context", Path: "something", Overwrite: true},
```

Comment:
Why should this be failing?

Reply:
Overwrite must be set on the fields doing the overwrite